modules/cert-manager-operator-configure-cpu-memory.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-customizing-api-fields.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-operator-configure-cpu-memory_{context}"]
+= Overriding CPU and memory limits for the {cert-manager-operator}
+
+After installing the {cert-manager-operator}, you can configure the CPU and memory limits for the {cert-manager-operator} from the cert-manager Operator API.
+
+.Prerequisites
+
+* You have access to the {product-title} cluster as a user with the `cluster-admin` role.
+* You have installed the {cert-manager-operator}.
+
+.Procedure
+
+. Check that the deployment of the cert-manager Operator is available by entering the following command:
++
+[source,terminal]
+----
+$ oc get deployment -n cert-manager-operator
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                       READY   UP-TO-DATE   AVAILABLE   AGE
+cert-manager-operator-controller-manager   1/1     1            1           51m
+----
+
+. Check the default resource configuration by entering the following command:
++
+[source,terminal]
+----
+$ oc get deployment -n cert-manager-operator -o yaml
+----
++
+.Example output
+[source,terminal]
+----
+...
+  metadata:
+    name: cert-manager-operator-controller-manager
+    namespace: cert-manager-operator
+...
+  spec:
+    template:
+      spec:
+        containers:
+        - name: kube-rbac-proxy
+          resources:
+            limits:
+              cpu: 500m
+              memory: 128Mi
+            requests:
+              cpu: 5m
+              memory: 64Mi
+...
+        - name: cert-manager-operator
+          resources:
+            requests:
+              cpu: 10m <1>
+              memory: 32Mi <2>
+...
+----
+<1> Specifies the CPU limit for the {cert-manager-operator}. The default value is `10m`.
+<2> Specifies the memory limit for the {cert-manager-operator}. The default value is `32Mi`.
+
+. To configure the CPU and memory limits for the {cert-manager-operator}, enter the following command:
++
+[source,terminal]
+----
+$ oc patch subscription openshift-cert-manager-operator -n cert-manager-operator --type=merge -p="
+spec:
+  config:
+    resources:
+      requests:
+        cpu: 20m
+        memory: 32Mi
```

Comment:
Good to add <1>, <2> placeholder(s) here I mean. And, yeah placeholder since they're all user customizable.
Additionally, you can add `limits:` section with similar cpu, memory; just letting users know that they can customize both requests and limits from this same way. If users want to omit either and keep only requests or only limits, that's supported too and we could mention that.